shotover-proxy/src/transforms/cassandra/peers_rewrite.rs

```diff
@@ -1,3 +1,4 @@
+use crate::frame::cassandra::CQLStatement;
```

Comment:
Can the changes in this file be reduced to the bare minimum to retain the existing functionality and then the improvements/fixes be moved to a follow up PR?
Theres a lot of complex changes going on in here and I don't feel comfortable reviewing it all at the same time.